{
  if (INIT_CAUSE_METHOD != null) {
    try {
      INIT_CAUSE_METHOD.invoke(throwable,new Object[]{cause});
    }
 catch (    Exception e) {
    }
  }
}
